### ISSUE TITLE
Choosing different fonts and colour schemes

### ISSUE THREAD

**Issue by jrttk** [jrttk/inko#2]
### Description

I'm looking forward to adding themes system to inko, so that user could be more comfortable writing with their favourite colour scheme and fonts !

ps. also dark mode included

**Comment by runyasak** [comment]
Did you mean like `font-family`? I will try to make font-family selection.

For theme and colour scheme, can you suggest me for some example or set of colour scheme that you prefer? 😁

**Comment by jrttk** [comment]
@runyasak Yes! I think the option to choose font-family and colour scheme is awesome and could make Inko more interesting. For the colour scheme, I think we could use the colour schemes from https://vscodethemes.com/ or https://iterm2colorschemes.com/ for references.
